hudi-spark-datasource/hudi-spark3-common/src/main/scala/org/apache/spark/sql/adapter/BaseSpark3Adapter.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.sql.adapter
 
-import org.apache.hudi.{AvroConversionUtils, DefaultSource, HoodiePartitionCDCFileGroupMapping, HoodiePartitionFileSliceMapping, Spark3HoodiePartitionCDCFileGroupMapping, Spark3HoodiePartitionFileSliceMapping}
+import org.apache.hudi.{AvroConversionUtils, DefaultSource, HoodiePartitionCDCFileGroupMapping, HoodiePartitionFileSliceMapping, HoodieSchemaConversionUtils, Spark3HoodiePartitionCDCFileGroupMapping, Spark3HoodiePartitionFileSliceMapping}
```

Comment:
Is this causing a runtime issue? How did the other PR pass without the import?

Reply:
<img width="1307" height="392" alt="Screenshot 2025-12-16 at 9 31 12 AM" src="https://github.com/user-attachments/assets/a21f3c93-d045-40b6-bdb7-3718782e399e" />
Actually not sure why this was added, since not being used. Will remove